### PR TITLE
Add CI status

### PR DESCRIPTION
**Pull request by skywall**

### CODE REVIEW

**Comment by thefuntasty-ops** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          11 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/thefuntasty/donut/blob/b6eab1ed95dbae9772d9cbce9bff86eb455634bb/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L13">library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L13</a> - Constructor parameter names should match the pattern: [a-z][A-Za-z\d]*</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/thefuntasty/donut/blob/b6eab1ed95dbae9772d9cbce9bff86eb455634bb/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L14">library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L14</a> - Constructor parameter names should match the pattern: [a-z][A-Za-z\d]*</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/thefuntasty/donut/blob/b6eab1ed95dbae9772d9cbce9bff86eb455634bb/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L15">library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L15</a> - Constructor parameter names should match the pattern: [a-z][A-Za-z\d]*</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/thefuntasty/donut/blob/b6eab1ed95dbae9772d9cbce9bff86eb455634bb/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L16">library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L16</a> - Constructor parameter names should match the pattern: [a-z][A-Za-z\d]*</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/thefuntasty/donut/blob/b6eab1ed95dbae9772d9cbce9bff86eb455634bb/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L17">library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L17</a> - Constructor parameter names should match the pattern: [a-z][A-Za-z\d]*</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/thefuntasty/donut/blob/b6eab1ed95dbae9772d9cbce9bff86eb455634bb/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L18">library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L18</a> - Constructor parameter names should match the pattern: [a-z][A-Za-z\d]*</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/thefuntasty/donut/blob/b6eab1ed95dbae9772d9cbce9bff86eb455634bb/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L19">library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L19</a> - Constructor parameter names should match the pattern: [a-z][A-Za-z\d]*</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/thefuntasty/donut/blob/b6eab1ed95dbae9772d9cbce9bff86eb455634bb/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L120">library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L120</a> - Parentheses in (pathLen.toDouble() * length) are unnecessary and can be replaced with: pathLen.toDouble() * length</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/thefuntasty/donut/blob/b6eab1ed95dbae9772d9cbce9bff86eb455634bb/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt#L52">library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt#L52</a> - Parentheses in (0f..1f) are unnecessary and can be replaced with: 0f..1f</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/thefuntasty/donut/blob/b6eab1ed95dbae9772d9cbce9bff86eb455634bb/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt#L228">library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt#L228</a> - Functions with exact one statement, the return statement, can be rewritten with ExpressionBodySyntax.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/thefuntasty/donut/blob/b6eab1ed95dbae9772d9cbce9bff86eb455634bb/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt#L255">library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt#L255</a> - it.animatedValue cannot be safely cast to Float.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: buildSrc/src/main/kotlin/com/thefuntasty/donut/BitriseLint.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.thefuntasty.donut
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.plugins.ExtraPropertiesExtension
+import org.gradle.kotlin.dsl.configure
+
+open class BitriseLint : DefaultTask() {
```
Comment (matejcikm) [inline buildSrc/src/main/kotlin/com/thefuntasty/donut/BitriseLint.kt]: `BitriseLint ` -> `LintCheck` ? 🤔

File: buildSrc/src/main/kotlin/com/thefuntasty/donut/BitriseLint.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    init {
+        group = "thefuntasty"
+
+        configure<ExtraPropertiesExtension> {
+            dependsOn("detekt")
+            project.subprojects.forEach {
+                dependsOn("${it.name}:ktlintCheck")
+            }
```
Comment (matejcikm) [inline buildSrc/src/main/kotlin/com/thefuntasty/donut/BitriseLint.kt]: + android lint?

Comment (skywall) [reply buildSrc/src/main/kotlin/com/thefuntasty/donut/BitriseLint.kt]: I have created separate issues for this https://github.com/thefuntasty/donut/issues/15

**Pull request merged**